tests/unit/type/list.tcl

```diff
@@ -289,6 +358,45 @@ start_server {
         assert_equal 0 [r exists target]
     }
 
+    test "BRPOPRPUSH with a client BLPOPing the target list" {
+        set rd [redis_deferring_client]
+        set rd2 [redis_deferring_client]
+        r del blist target
+        $rd2 blpop target 0
+        $rd brpoprpush blist target 0
+        after 1000
```

Comment:
for these 4 tests, again, let's take this opportunity to replace the `after 1000` with a `wait_for_condition` for `[s blocked_clients] == 2`

Reply:
Done.